modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/EditApp.es.js

```diff
@@ -89,19 +90,19 @@ export default ({
 
 			<Loading isLoading={isLoading}>
 				<EditAppContext.Provider value={{dispatch, state}}>
-					<div className="container-fluid container-fluid-max-lg mt-4">
+					<ClayLayout.ContainerFluid className="mt-4" size="lg">
 						<div className="card card-root mb-0 shadowless-card">
 							<EditAppHeader />
 
 							<div className="card-body p-0 shadowless-card-body">
-								<div className="autofit-row">
-									<div className="col-md-12">
+								<ClayLayout.Row>
+									<ClayLayout.Col md="12">
```

Comment:
as above "what happen if we remove the `md`?"

Reply:
In this case, probably we can use only a col and remove `md="12"`